conanfile.py

```diff
@@ -9,7 +9,7 @@
 
 class SISLConan(ConanFile):
     name = "sisl"
-    version = "11.1.3"
+    version = "11.1.4"
```

Comment:
Technically this requires a MAJOR update to 12 since the methods for a public class was changed to be external linkage versus a inlined (can't link code that was built with 11.1.3 to code that is using 11.1.4) without unexpected behavior.
Assuming the the project in the middle (e.g. homestore) doesn't use this interface we're okay...but because *we* know that. Do we or are we just going to bump things iteratively everywhere?

Reply:
Homestore does use the header `generic_service.hpp` which is changed to external linkage. Is the behavior unexpected even if we rebuild stuff down stream? I did not want to bump major version just for this, I had to change the linkage as I was having difficulties including the `utils.hpp` file defined in the src folder.

Reply:
No; it'll work fine if we build nuraft_mesg with the updated sisl before building homestore (since they both use it right?). If only one of them is using it then there is zero concern; it would just be if one constructed some object with the inline and passed it to other code compiled with the constructor having different behavior. This is just true of changing _any_ inline defined in a public API.